Godeps/_workspace/src/github.com/Clever/gearadmin/.drone.yml

```diff
@@ -0,0 +1,13 @@
+image: go1.2
```

Comment:
This line indicates gearadmin (a dependency) is tested against go 1.2. Godeps.json indicates that gearman-load-logger is built against go 1.3.